config/v1alpha1/types_cluster_monitoring.go

```diff
@@ -89,6 +89,11 @@ type ClusterMonitoringSpec struct {
 	// The current default value is `DefaultConfig`.
 	// +optional
 	AlertmanagerConfig *AlertmanagerConfig `json:"alertmanagerConfig,omitempty"`
+	// metricsServerConfig metricsServerConfig defines the configuration for the Kubernetes Metrics Server.
+	// metricsServerConfig is optional.
+	// When omitted, this means no opinion and the platform is left to choose a reasonable default, that is subject to change over time.
+	// +optional
+	MetricsServerConfig *MetricsServerConfig `json:"metricsServerConfig,omitempty"`
```

Comment:
Is the metrics server always deployed or is this something users can opt out of deploying like Alertmanager?

Reply:
it is always deployed